packages/react-scripts/template-ef/src/server.js

```diff
@@ -0,0 +1,10 @@
+var express = require("express");
+var path = require("path");
+var app = express();
+
+// React App
+app.use(express.static(__dirname + "/build"));
+
+const port = process.env.PORT || 3000;
+app.listen(port);
+console.log("Listening on port " + port)
```

Comment:
Is this a placeholder for Frontier server or just for Heroku? Heroku has a CRA buildpack if that's all we need

Reply:
just a placeholder for now. Tyler will be beefing this file up with snow with his CRA w/ Snow stuff.